appuio/redis/README.md

```diff
@@ -1,6 +1,6 @@
 # redis
```

Comment:
I forgot to mention: Please add the `README.gotmpl.md` file like the other charts do, e.g. https://raw.githubusercontent.com/appuio/charts/master/appuio/secret/README.gotmpl.md

That will alter the README slightly